packages/jest-config/src/vendor/jsonlint.js

```diff
@@ -287,7 +287,6 @@ var jsonlint = (function() {
         preErrorSymbol,
         state,
         action,
-        a,
```

Comment:
this is a vendor file, it should be left alone

Reply:
👍